kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -807,9 +807,10 @@ These are just used by the other operators for shuffling local execution state a
            <acctID> ACCT  </acctID>
            <code>   CODE  </code>
            <nonce>  NONCE </nonce>
+           <storage> STORAGE </storage>
            ...
          </account>
-      requires CODE =/=K .Bytes orBool NONCE =/=Int 0
+      requires CODE =/=K .Bytes orBool NONCE =/=Int 0 orBool STORAGE =/=K .Map
 
     rule <k> #newExistingAccount ACCT => .K ... </k>
```

Comment:
This rule ought to be deleted. It overlaps now with the one above.

Reply:
I'm not sure about it. Deleting it make 4 tests to fail:
- `Pyspecs/cancun/eip6780_selfdestruct/calling_from_new_contract_to_pre_existing_contract.json`
- `stRevertTest/RevertPrefoundEmpty_Paris.json`
- `stCreateTest/TransactionCollisionToEmpty2.json`
- `stRevertTest/RevertPrefoundOOG.json`